global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.1.416"
```

Comment:
It looks like the build is failing as the agent is on version:
        dotnet-version: 3.1.301
 We could update the build agent if you need this version?

Reply:
I have removed this file. 
Didn't intend to update this